app/scripts/controllers/transactions/index.test.js

```diff
@@ -2991,4 +2991,37 @@ describe('Transaction Controller', function () {
       assert.equal(result.type, TransactionType.simpleSend);
     });
   });
+
+  describe('addInitialTransactionApprovals', function () {
+    it('adds unapprovedTxs as approvals', async function () {
+      const txId = '1';
+      txController.addTransaction(
```

Comment:
Is it safer to add two transactions to verify we're iterating and support multiple pending approvals?

Reply:
Done